components/molecule/selectPopover/src/_settings.scss

```diff
@@ -1,4 +1,9 @@
-$bdrs-select-popover: 16px !default;
+$bdrs-select-popover: 20px !default;
```

Comment:
If we change the default value, this could be a breaking change if somebody is using it relying on the default value. Couldn't you just change this value in your theme?

Reply:
This change is necessary because with the new sizes `16px` is not enought. 
Anyway, nobody is using this component yet (only us).